industry/products.md

```diff
@@ -12,16 +12,19 @@ They vary by:
 - accuracy
 - price
 - customisation options
-- data confidentiality
-
+- [data confidentiality](data-confidentiality.md)
 and other features, as well as underlying architecture.
 
+Some providers offer ready-to-use solutions that can be integrated immediately.  These usually support many language pairs and their infrastructure is scalable.  An alternative is to train a model from scratch — or to use a pre-trained model — and deploy and maintain it.
 
-An advantage of using a machine translation provider is that the service is already working and can be integrated immediately. An alternative to using a machine translation provider is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
-
-Unlike raw models, production machine translation systems usually provide language identification and sentence segmentation.
+Unlike raw models, ready-to-use solution usually include:
```

Comment:
Should be plural.

Reply:
What do you think about "usually"?  Is it really true for all the stuff we list below?

Maybe "often"?

I think it's more true/evergreen to say.  "Raw models usually do not include the features included in many production systems:"